docs/docsite/rst/network/user_guide/network_working_with_command_output.rst

```diff
@@ -63,3 +66,58 @@ The ``wait_for`` argument must always start with result and then the
 command index in ``[]``, where ``0`` is the first command in the commands list,
 ``1`` is the second command, ``2`` is the third and so on.
 
+
+Handling prompts in network modules
+===================================
+
+Network devices may require that you answer a prompt before performing a change on the device. Individual network modules such as :ref:`ios_command <ios_command_module>` and :ref:`nxos_command <nxos_command_module>` can handle this with a ``prompt`` parameter.
+
+You can also use the :ref:`cli_command <cli_command_module>` to handle multiple prompts.
+
+.. code-block:: yaml
+
+  ---
+  - name: multiple prompt, multiple answer (mandatory check for all prompts)
+    cli_command:
+      command: "copy sftp sftp://user@host//user/test.img"
+      check_all: True
+      prompt:
+        - "Confirm download operation"
+        - "Password"
+        - "Do you want to change that to the standby image"
+      answer:
+        - 'y'
+        - <password>
+        - 'y'
+
+You must list the prompt and the answers in the same order (that is, prompt[0] is answered by answer[0]).
+
+.. note::
+
+	``prompt`` is a Python regex. If you add special characters such as ``?`` in the ``prompt`` value, the prompt won't match and you will get a timeout. To avoid this, ensure that the ``prompt`` value is a Python regex that matches the actual device prompt. Any special characters must be handled correctly in the ``prompt`` regex.
```

Comment:
_Personally_, I would put this paragraph after https://github.com/ansible/ansible/pull/66584/files#diff-d18c6126819d945c63be40b4b8126fd8R74 so that it's among the first things mentioned in this section, and then add a sentence at the end of https://github.com/ansible/ansible/pull/66584/files#diff-d18c6126819d945c63be40b4b8126fd8R101 reiterating that prompt is a regex and pointing out that the following example includes an escaped question mark for that reason.

Reply:
done, thanks!